cmd/podman/rmi.go

```diff
@@ -63,13 +66,21 @@ func rmiCmd(c *cli.Context) error {
 	for _, arg := range imagesToDelete {
 		image, err := runtime.GetImage(arg)
 		if err != nil {
-			return errors.Wrapf(err, "could not get image %q", arg)
+			if lastError != nil {
+				fmt.Fprintln(os.Stderr, lastError)
+			}
+			lastError = errors.Wrapf(err, "could not get image %q", arg)
+			continue
 		}
 		id, err := runtime.RemoveImage(image, c.Bool("force"))
 		if err != nil {
-			return errors.Wrapf(err, "error removing image %q", id)
+			if lastError != nil {
+				fmt.Fprintln(os.Stderr, lastError)
+			}
+			lastError = errors.Wrapf(err, "failed to remove image")
```

Comment:
Should put this in an else, as it is we overwrite lastError unconditionally now

Reply:
but isn't that what we want to do? If an error already occurred, print it out and replace lastError with the new error that occurred?
I followed what was in rm.go, unless it is wrong there as well.

Reply:
On further review, this does look correct - I was assuming we were printing err instead of lastError. As written this should be fine.